### ISSUE TITLE
Docstring for some modules does not look right

### ISSUE THREAD

**Issue by karthiknadig** [pappasam/jedi-language-server#68]
Doc string for `sys` module with `markdown` style comes back with strange markup. This might be a general problem with docstrings. This is just part of it, that we get back from jedi-language-server:
```
\_enablelegacywindowsfsencoding -- [Windows only]
**stdin** -- the original stdin; don't touch!
**stdout** -- the original stdout; don't touch!
**stderr** -- the original stderr; don't touch!
**displayhook** -- the original displayhook; don't touch!
**excepthook** -- the original excepthook; don't touch!
```
Actual:
![image](https://user-images.githubusercontent.com/3840081/107559378-dea8d000-6b90-11eb-97d4-612dcae51de9.png)

Expected:
![image](https://user-images.githubusercontent.com/3840081/107559129-870a6480-6b90-11eb-8e80-fb2e0e901a33.png)

**Comment by pappasam** [comment]
@karthiknadig please let me know if this is resolved from your end on the latest release!

**Comment by bollwyvl** [comment]
Also, it looks like the docstring-to-markdown pin might be too aggressive... I'm trying to ship the latest version on [conda-forge](https://github.com/conda-forge/jedi-language-server-feedstock/pull/5) and am seeing:

```
jedi-language-server 0.25.2 has requirement docstring-to-markdown<0.5,>=0.4, but you have docstring-to-markdown 0.5.
```

I'll build `0.4` to satisfy the pin, but I'm not sure if that is the intent... looks like some features on `0.5` might fix some issues.

**Comment by krassowski** [comment]
I can promise no breaking API changes prior to 1.0 if this helps.

**Comment by bollwyvl** [comment]
`0.25.2` is up, and ready to go... happy to help with whatever to keep the pipeline going!

**Comment by pappasam** [comment]
@bollwyvl thanks for calling that out, fixed in `0.25.3`. Per @krassowski , I've relaxed the version constraint to allow anything below `1.0` for `docstring-to-markdown`.

**Comment by pappasam** [comment]
@karthiknadig I believe this specific issue involving the `sys` module is resolved. If you notice anything else that's wonky, please open a new issue.
